test/CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ add_test(
 set_tests_properties(oqs_groups
     PROPERTIES ENVIRONMENT "OPENSSL_MODULES=${CMAKE_BINARY_DIR}/lib"
 )
-add_executable(oqs_test_groups oqs_test_groups.c test_common.c ${OPENSSL_BLDTOP}/test/helpers/ssltestlib.c)
+add_executable(oqs_test_groups oqs_test_groups.c test_common.c tlstest_helpers.c)
 # e_os.h resides in top-level for OSSL3:
```

Comment:
Any OPENSSL_BLDTOP dependency here should be (possible to be?) removed here, too

Reply:
Fixed